From 5b4df09b709a88cf19bd4c9d6ff01978088bfb2e Mon Sep 17 00:00:00 2001
From: Alexandre Frade <kernel@xanmod.org>
Date: Fri, 30 May 2025 19:58:58 +0000
Subject: [PATCH] XANMOD: mm/vmscan: Reduce amount of swapping

Signed-off-by: Alexandre Frade <kernel@xanmod.org>
---
 mm/vmscan.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/mm/vmscan.c b/mm/vmscan.c
index ee3efc621..01027de6b 100644
--- a/mm/vmscan.c
+++ b/mm/vmscan.c
@@ -196,7 +196,7 @@ struct scan_control {
 /*
  * From 0 .. MAX_SWAPPINESS.  Higher means more swappy.
  */
-int vm_swappiness = 60;
+int vm_swappiness = 10;

 static int sc_swappiness(struct scan_control *sc, struct mem_cgroup *memcg)
 {
